adapters/timer.c

```diff
@@ -77,3 +77,21 @@ void timer_destroy(TIMER_HANDLE timer)
         free(timer);
     }
 }
+
+static LARGE_INTEGER g_freq;
+static volatile LONG g_timer_state = 0; /*0 - not "created", 1 - "created", "2" - creating*/
```

Comment:
>t "created", 1 - "created", "2" - creating*/ [](start = 48, length = 44)

I'd do an enum for these states, easier to figure in the code what the states are rather than looking at numbers :-).

Reply:
I initially had an enum, then I decided I liked the approach with "0,1,2" better. They are all confined to 3 lines of code.